client/src/components/EmailField/EmailField.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import styles from './EmailField.module.scss';
+import TextField from '../TextField/TextField.tsx';
+
+/** 
+ * EmailField Component
+ * Represents a form field for resetting password with email input and basic validation.
+ * This component provides a text field for users to enter their email address and displays
+ * an error message if the entered email format is invalid
+ * 
+*/
+export default function ResetPassword(): JSX.Element {
```

Comment:
The component and whole directory will be renamed to ResetPassword since I was confused on exactly what I needed to do.